lib/PromptSession.ts

```diff
@@ -188,101 +189,120 @@ export class PromptSession {
-				const component = await inquirer.prompt({
-					type: "list",
-					name: "component",
-					message: "Choose a component:",
-					choices: this.addSeparators(componentNames)
-				});
+					const chosenGroupName = group["componentGroup"];
 
-				const pickedComponent = framework.getComponentByName(component["component"]);
+					if (chosenGroupName === this.WIZARD_BACK_OPTION) {
+						addComponentCompleted = true;
+						continue;
+					}
 
-				GoogleAnalytics.post({
-					t: "event",
-					ec: "$ig wizard",
-					el: "Choose a component",
-					ea: `component: ${component["component"]}`,
-					cd6: pickedComponent.name
-				});
+					GoogleAnalytics.post({
+						t: "event",
+						ec: "$ig wizard",
+						el: "Choose a group",
+						ea: `component group: ${chosenGroupName}`,
+						cd5: group["componentGroup"]
+					});
 
-				// runTemplateCollection (item: Template[])
-				//TODO refactor
-				const templates: Template[] = pickedComponent.templates;
-				if (templates.length === 1) {
-					//get the only one template
-					selectedTemplate = templates[0];
-				} else {
-					const templateNames = templates.map(x => x.name);
-					const template = await inquirer.prompt({
+					const componentNames = framework.getComponentNamesByGroup(chosenGroupName);
+					componentNames.push(this.WIZARD_BACK_OPTION);
+					const component = await inquirer.prompt({
 						type: "list",
-						name: "template",
-						message: "Choose one:",
-						choices: this.addSeparators(templateNames)
+						name: "component",
+						message: "Choose a component:",
+						choices: this.addSeparators(componentNames)
 					});
 
-					selectedTemplate = templates.find((value, i, obj) => {
-						return value.name === template["template"];
-					});
+					const chosenComponentName = component["component"];
+
+					if (chosenComponentName === this.WIZARD_BACK_OPTION) {
+						continue;
+					}
+
+					addComponentCompleted = true;
+					const pickedComponent = framework.getComponentByName(chosenComponentName);
 
 					GoogleAnalytics.post({
 						t: "event",
 						ec: "$ig wizard",
-						el: "Choose one (template):",
-						ea: `template: ${template["template"]}`,
-						cd7: selectedTemplate.id
+						el: "Choose a component",
+						ea: `component: ${chosenComponentName}`,
+						cd6: pickedComponent.name
 					});
-				}
-				if (selectedTemplate) {
-					let success = false;
-					while (!success) {
-						templateName = await inquirer.prompt({
-							type: "input",
-							name: "name",
-							message: "Name your component:",
-							default: selectedTemplate.name
+
+					// runTemplateCollection (item: Template[])
+					//TODO refactor
+					const templates: Template[] = pickedComponent.templates;
+					if (templates.length === 1) {
+						//get the only one template
+						selectedTemplate = templates[0];
+					} else {
+						const templateNames = templates.map(x => x.name);
+						const template = await inquirer.prompt({
```

Comment:
There should also be an option to go back from here ( Components > Grid > List of grid templates)